src/Fable.Core/RELEASE_NOTES.md

```diff
@@ -1,3 +1,16 @@
+### 4.0.0-snake-island-alpha-007
+
+* Python, import fixes, @dbrattli
+* Rust, records and Unions now are correctly unwrapped when Struct attribute is used to tag as value, alexswan10k
+* Rust, support struct anonymous records, @ncave
+* Rust, added support for struct tuples, @ncave
+* Rust, fixed struct constructors, @ncave
+* Rust, made Async optional, @ncave
+* Rust, added missing type import and slices, @ncave
+* Rust, comparison updates, @ncave
+* Make CancellationTokenSource implement IDisposable, @alfonsogarciacaro
+* Fixed Array.compareWith issue, @alfonsogarciacaro
+
```

Comment:
@dbrattli Shouldn't these release notes go into `src/Fable.Cli/RELEASE_NOTES.md`?

Reply:
@ncave Yes, your are right. Sorry about the mistake. I'll try to clean it up.